server/src/routes/api/v1/dogsRouter.js

```diff
@@ -41,4 +44,24 @@ dogsRouter.get("/:id", async (req, res) => {
 
   }
 })
+
+dogsRouter.post("/:dogId", async (req, res) => {
```

Comment:
I'm not sure what the difference is between this endpoint and the POST one above it? I see - this endpoint is for creating a review for a dog. As such, this should instead be a nested router. We are not interacting with the `dog` resource as this router would suggest. but instead want to make a `dogReviews` nested router for better organization for this specific POST reviews endpoint. With that, we want to make sure we're following RESTful API endpoints, so the POST path would be `/api/v1/dogs/:dogId/reviews` since it is the _reviews_ resource we're trying to create (not a dog, or dogId)